bundler/lib/bundler/uri_credentials_filter.rb

```diff
@@ -16,7 +16,7 @@ def credential_filtered_uri(uri_to_anonymize)
 
       if uri.userinfo
         # oauth authentication
-        if uri.password == "x-oauth-basic" || uri.password == "x"
+        if uri.password == "x-oauth-basic" || uri.password.nil?
```

Comment:
Removing the "x" special case is actually unrelated to this PR, so I'll remove it, but I wondering where that comes from?